protractor/local.config.ts

```diff
@@ -1,10 +1,12 @@
 import { Config, browser } from "protractor";
+import { reporter } from "./helpers/reporter";
 
 export const config: Config = {
   framework: "jasmine",
   specs: ["../test/google.spec.js"],
   seleniumAddress: "http://localhost:4444/wd/hub",
   onPrepare: () => {
     browser.ignoreSynchronization = true;
+    reporter();
```

Comment:
reporter should go outside of the onPrepare function

Reply:
When i put the reporter outside of the onPrepare function an error is generated

Reply:
yes, you're right, I'm sorry, got confused.